cmd/kubectl-gadget/seccompadvisor.go

```diff
@@ -0,0 +1,185 @@
+// Copyright 2019-2021 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/kinvolk/inspektor-gadget/cmd/kubectl-gadget/utils"
+	gadgetv1alpha1 "github.com/kinvolk/inspektor-gadget/pkg/api/v1alpha1"
+)
+
+var seccompAdvisorCmd = &cobra.Command{
+	Use:   "seccomp-advisor",
+	Short: "Generate seccomp policies based on recorded syscalls activity",
+}
+
+var seccompAdvisorStartCmd = &cobra.Command{
+	Use:          "start",
+	Short:        "Start to monitor the system calls",
+	RunE:         runSeccompAdvisorStart,
+	SilenceUsage: true,
+}
+
+var seccompAdvisorStopCmd = &cobra.Command{
+	Use:          "stop",
+	Short:        "Stop monitoring and report the policies",
+	RunE:         runSeccompAdvisorStop,
+	SilenceUsage: true,
+}
+
+var seccompAdvisorListCmd = &cobra.Command{
+	Use:          "list",
+	Short:        "List existing seccomp traces",
+	RunE:         runSeccompAdvisorList,
+	SilenceUsage: true,
+}
+
+var (
+	outputMode         string
+	seccompProfileName string
+)
+
+func init() {
+	// Add generic information.
+	rootCmd.AddCommand(seccompAdvisorCmd)
+	utils.AddCommonFlags(seccompAdvisorCmd, &params)
+
+	seccompAdvisorCmd.AddCommand(seccompAdvisorStartCmd)
+	seccompAdvisorStartCmd.PersistentFlags().StringVarP(&outputMode,
+		"output-mode", "m",
+		"terminal",
+		"The trace output mode, possibles values are terminal (default) and seccomp-profile.")
+	seccompAdvisorStartCmd.PersistentFlags().StringVar(&seccompProfileName,
+		"seccomp-profile-name", "",
+		"The seccomp profile which will be created when using seccomp-profile as value for --output-mode.\nYou can precise the namespace under which the profile will be created by calling it: namespace/profile-name.")
```

Comment:
What about using a more concise language like:

```suggestion
		"Name of the seccomp profile to be created when using --output-mode=seccomp-profile.\n"+
			"Namespace can be especified by using namespace/profile-name")
```